attributes/default.rb

```diff
@@ -71,3 +71,5 @@
   else
     []
   end
+
+default['graphite']['pyenv']['python_version'] = '2.7.17'
```

Comment:
We don’t want to support defat versioning like this anymore. It should be a default on the resource instead

Reply:
I moved the python install to its own resource, how about that implementation?